packages/ui/src/accounts/modals/TransferModal/TransactionInfo.tsx

```diff
@@ -0,0 +1,42 @@
+import BN from 'bn.js'
+import React from 'react'
+
+import { Help } from '../../../common/components/Help'
+import { BalanceInfoNarrow, InfoTitle, InfoValue } from '../../../common/components/Modal'
+import { TokenValue } from '../../../common/components/typography'
+
+interface TransactionInfoProps {
```

Comment:
Because this component is using only "common" components, it should land somwhere in "src/common/components"